StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheet+APITest.swift

```diff
@@ -12,8 +12,8 @@ import XCTest
 @testable@_spi(STP) import StripeCore
 @testable@_spi(STP) import StripeCoreTestUtils
 @testable@_spi(STP) import StripePayments
-@testable@_spi(STP) import StripePaymentsTestUtils
 @testable@_spi(STP) import StripePaymentSheet
+@testable@_spi(STP) import StripePaymentsTestUtils
```

Comment:
linter I guess?

Reply:
Hm it's not supposed to touch files you didn't modify